app/controllers/history_items_controller.rb

```diff
@@ -93,6 +95,14 @@ def history_item_params
       params.require(:history_item).permit(:taxt, :rank)
     end
 
+    def redirect_back_url
+      params[:redirect_back_url].presence
+    end
```

Comment:
For https://github.com/calacademy-research/antcat-issues/issues/72#issuecomment-733705062